core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -781,6 +792,7 @@ private void handleLocaleCheck() {
     tabCallback = null;
     hideBackToTopTimer.cancel();
     hideBackToTopTimer = null;
+    webViewList.clear();
```

Comment:
```
    super.onDestroyView();
    safeDispose();
    super.onDestroy();
```
why are we calling the super of another method, that seems like an error

also why is webViewList.clear now needed